README.md

```diff
@@ -58,6 +58,21 @@ the `# Anopheles gambiae` comment:
 - accession: XXX_000000000.0
 ```
 
+### Updating workflows
+
+Run
+
+```shell
+python catalog/source/iwc_manifest_to_workflow_yaml.py --exclude-other
```

Comment:
looks like we have a new helper in package.json, but readme isnt updated to reflect this?